### PR TITLE
Fine tweak of docs

### PR DESCRIPTION
**Pull request by shaojingli**
Following https://github.com/uber/deck.gl/issues/496

### CODE REVIEW

File: docs/api-reference/attribute-manager.md [hunk 1]
```diff
@@ -13,29 +13,12 @@ Limitations:
   There are currently no provisions for only invalidating a range of
   indices in an attribute.
 
-## Static Methods
```
Comment (ibgreen) [inline docs/api-reference/attribute-manager.md]: `setDefaultLogFunctions ` is actually public - Just added it a couple of days ago. My mistake...

File: docs/api-reference/attribute-manager.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # AttributeManager Class
 
-The `AttributeManager` class automated attribute allocation and updates.
+The [`AttributeManager`](https://github.com/uber/deck.gl/blob/master/src/lib/attribute-manager.js) class provides automated attribute allocations and updates.
```
Comment (ibgreen) [inline docs/api-reference/attribute-manager.md]: Just realized: For source links, should we link to `master` or to `4.0-release`?

File: docs/api-reference/base-layer.md [hunk 3]
```diff
@@ -26,10 +26,11 @@ of deck.gl, a check will be in place to enforce this
 
 ## Constructor
 
-- `props` (Object) - Layer properties.
-
 ### Basic Properties
 
+##### `props` (Object)
```
Comment (ibgreen) [inline docs/api-reference/base-layer.md]: This looks strange, I think this was intended to be the parameter list of the constructor

I think it is better to move this back and add a heading `Parameters`:

## Constructor

Initializes a new `Layer` instance.

Parameters:
- `props` (Object) - Layer properties.

File: docs/api-reference/attribute-manager.md (hunk 2 above)
Comment (ibgreen) [inline docs/api-reference/attribute-manager.md]: Nit: When done this way, it is not clear what is a doc link and what is a source link.

I thought having a link that actually says `Source` (perhaps at the bottom of the page) would be cleaner. See for instance how THREE.js does it https://threejs.org/docs/api/math/Box2.html

File: docs/layers/arc-layer.md [hunk 4]
```diff
@@ -6,10 +6,6 @@
 
 # ArcLayer
 
-The Arc Layer renders raised arcs joining pairs of source and target points,
```
Comment (ibgreen) [inline docs/layers/arc-layer.md]: Just to clarify, when I commented about "source link" should be separated from the paragraph, I meant the link to the original source code in the repo.
I.e this piece:
Source](https://github.com/uber/deck.gl/tree/master/src/layers/core/arc-layer)

File: docs/api-reference/attribute-manager.md (hunk 2 above)
Comment (shaojingli) [inline docs/api-reference/attribute-manager.md]: You are right. We should do 4.0-release. I'll branch 4.0-release after all outstanding PRs got landed today

**Comment by shaojingli** [comment]
All links to source code now points to the 4.0-release branch instead of the master branch.

@Pessimistress When I start the demo site locally, all the links to the source code doesn't open in the same tab. The links are good as they can be opened in another tab without problems. Do you have any ideas why this happens?

screen shot 2017-04-03 at 2 35 41 pm

**Pull request merged**